app/views/hub/clients/show.html.erb

```diff
@@ -148,7 +148,7 @@
                 </div>
               <% end %>
               <% unless @client.intake.drop_off? %>
-                <% tax_return = @client.tax_returns.find_by(year: Rails.application.config.current_tax_year) %>
+                <% tax_return = @client.tax_returns.find_by(year: TaxReturn.current_tax_year) %>
```

Comment:
It seems to me this that this could be a call to `@client.intake.default_tax_return`, which I would prefer for consistency. Definitely not a blocker.